enroll_me/event_handlers.py

```diff
@@ -145,8 +145,11 @@
         await gh_api.patch(
             reaction_comment['url'],
             data={
-                'body':
-                f'@{comment_author} Here\'s your repo: '
-                f'{repo["html_url"]}{bot_sign}',
+                'body': (
```

Comment:
Braces aren't actually needed because the parser can recognize a string w/o them...

Reply:
Yep. I'm using them to make it clearer to, visually, know where the string starts and ends. :)